providers/aws/components/ecs/setup.ftl

```diff
@@ -1089,166 +1089,88 @@
+                            "awsvpcConfiguration" : {
+                                "Subnets" : subnets,
+                                "AssignPublicIp" : publicRouteTable?then("ENABLE", "DISABLED")
+                            }
+                        }
+                    ) +
+                    attributeIfTrue(
+                        "LaunchType",
+                        engine == "fargate",
+                        "FARGATE"
+                    )]
 
-                        [#if deploymentSubsetRequired("prologue", false)]
-                            [@addToDefaultBashScriptOutput
-                                content=
-                                    [
-                                        " case $\{STACK_OPERATION} in",
-                                        "   delete)",
-                                        "       # Manage Scheduled Event",
-                                        "       info \"Deleting Scheduled Task...\"",
-                                        "       delete_cloudwatch_event" +
-                                        "       \"" + region + "\" " +
-                                        "       \"" + scheduleRuleId + "\" " +
-                                        "       \"true\" || return $?",
-                                        "       ;;",
-                                        " esac"
-                                    ]
-                            /]
-                        [/#if]
+                    [@createScheduleEventRule
+                        id=scheduleRuleId
+                        enabled=scheduleEnabled
+                        scheduleExpression=schedule.Expression
+                        targetParameters=targetParameters
+                        dependencies=fnId
+                    /]
 
-                    [#else]
-                        [#if deploymentSubsetRequired("ecs", true) ]
-                            [#local targetParameters = {
-                                "Arn" : getExistingReference(ecsId, ARN_ATTRIBUTE_TYPE),
-                                "Id" : taskId,
-                                "EcsParameters" : {
-                                    "TaskCount" : schedule.TaskCount,
-                                    "TaskDefinitionArn" : getReference(taskId, ARN_ATTRIBUTE_TYPE)
-                                },
-                                "RoleArn" : getReference(scheduleTaskRoleId, ARN_ATTRIBUTE_TYPE)
-                            }]
-
-                            [@createScheduleEventRule
-                                id=scheduleRuleId
-                                enabled=scheduleEnabled
-                                scheduleExpression=schedule.Expression
-                                targetParameters=targetParameters
-                                dependencies=fnId
-                            /]
-                        [/#if]
+                    [#if cliCleanUpScheduleRuleId?has_content ]
```

Comment:
Should that be if it doesn't have content? I.e. we haven't cleaned it up before? 
Also it will need an additional condition to see if the task already exists. Otherwise on a new task deployment the rule could end up removing the new scheduled event.

Reply:
@roleyfoley `cliCleanUpScheduleRuleId` has content when we set the pseudo stack output variable with the "cleanup" postfix - https://github.com/codeontap/gen3/pull/1009/files/04b8881d936289bb2ef79bb30cb4ec1eedd7c512#diff-48e25ae413e06c838128cd60d543fcdfR1098
then we re-generate the pseudo stack output file and set this variable to the empty string. If delete fails, the stack output file will not be re-generated.